### DISCUSSION TITLE
Height, width and background properties not working with new update v3.2

### DISCUSSION THREAD

**Discussion by Jae-Kae** [tailwindlabs/tailwindcss#9624] [Help]
After Oct 19th my height, width and background settings are no longer working.

 <div>
        <div>
          <div className="text-black w-10 h-96 bg-black">
            <p>Hello</p>
          </div>
        </div>
  </div>
      
  does not change the width or height or background at all.
  
  Anyone else experiencing this with the new update and know how to fix it?

**Comment by wongjn** [comment]
Seems to be fine for me, and also fine on Tailwind Play. Is it not working for a particular project for you? Is there a remotely-hosted git repository that you could provide that produces your issue?
